modules/promotion/src/CouponOrderProcessor.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\commerce_promotion;
+
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\commerce_order\OrderProcessorInterface;
+use Drupal\commerce_promotion\Plugin\Commerce\PromotionOffer\PromotionOfferInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Applies promotions to orders during the order refresh process.
```

Comment:
Wrong docblock. Let's do "Applies coupon promotions to orders during the order refresh process." and a @see to the PromotionOrderProcessor. We also want PromotionOrderProcessor to have a @see to this one.